File: API/ios-tms.Core/Domain/Quote.cs
using System;
namespace iOS_TMS.Core.Domain;

public class Quote : EntityBase
{
    public string Origin { get; set; }
    public string Destination { get; set; }
    public int Items { get; set; }
    public decimal Weight { get; set; }
    public DateTime PickupDate { get; set; }
    public bool IsDeleted { get; set; }

    public List<QuoteRate> QuoteRates { get; set; }
}

File: App/ios-tms/Models/Quote.swift
//
//  Quote.swift
//  ios-tms
//
//  Created by Kyle Morton on 9/13/22.
//

import Foundation

struct Quote : Identifiable, Codable {

    var id: Int
    var origin: String
    var destination: String
    var items: Int
    var weight: Decimal
    var lowestRate: Decimal
    var lowestRateFormatted: String
    var highestRate: Decimal
    var highestRateFormatted: String
    var rateCount: Int
    
    init(id: Int, origin: String, destination: String, items: Int, weight: Decimal, lowestRate: Decimal, lowestRateFormatted: String, highestRate: Decimal, highestRateFormatted: String, rateCount: Int) {
        self.id = id
        self.origin = origin
        self.destination = destination
        self.items = items
        self.weight = weight
        self.lowestRate = lowestRate
        self.lowestRateFormatted = lowestRateFormatted
        self.highestRate = highestRate
        self.highestRateFormatted = highestRateFormatted
        self.rateCount = rateCount
    }
    
}
